app/controllers/catalog_controller.rb

```diff
@@ -136,9 +196,12 @@ def self.modified_field
     # or can be specified manually to be different.
     #
     # This one uses all the defaults set by the solr request handler. Which
-    # solr request handler? The one set in config[:default_solr_parameters][:qt],
-    # since we aren't specifying it otherwise.
-    config.add_search_field('all_fields', label: 'All Fields', include_in_advanced_search: false) do |field|
+    # solr request handler? The one set in
+    # config[:default_solr_parameters][:qt], since we aren't specifying it
+    # otherwise.
+    config.add_search_field('all_fields',
+        label: 'All Fields',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.